hardware/OTGWBase.cpp

```diff
@@ -302,6 +302,7 @@ void OTGWBase::SetSetpoint(const int idx, const float temp)
 		_log.Log(LOG_STATUS, "OTGW: Setting Heating SetPoint to: %.1f", temp);
 		sprintf(szCmd, "SW=%.1f\r\n", temp);
 		WriteInt((const unsigned char*)&szCmd, (const unsigned char)strlen(szCmd));
+                UpdateSetPointSensor(idx, temp, "DHW Setpoint");
```

Comment:
this should be updated automatically in the next poll... but if you can not wait that long, you want to do this for the other set points as well ?